benchmarks/benchmarks.py

```diff
@@ -16,3 +16,41 @@ def time_distance(self):
 
     def time_intersection(self):
         pygeos.intersection(self.points, self.polygon)
+
+
+class IOSuite:
+    """Benchmarks I/O operations (WKT and WKB) on a set of 100000 polygons"""
+    def setup(self):
+        self.to_write = pygeos.polygons(np.random.random((100000, 2)))
```

Comment:
Note that this is not actually 10000 polygons, but a single polygon of 10000 vertices. I think it is better to actually construct 10000 polygons (``pygeos.polygons(np.random.random((10000, 3, 2)))`` to ensure we benchmark pygeos' overhead, and not just the GEOS operation

Reply:
Whoops. I'll fix that up.